heroku.xml

```diff
@@ -0,0 +1,3 @@
+build:
+  docker:
+    web: Dockerfile
```

Comment:
The $ shot right here; damn you Heroku; way to push through and find the solution!